### PR TITLE
WT-9922 Define new debug verbosity levels

### PR DESCRIPTION
**Pull request by korteland**
This is the same PR as the last one, except that I added `#define WT_VERBOSE_DEBUG WT_VERBOSE_DEBUG_1` so that my changes don't break compatibility while the server changes aren't done. This means I can merge things into develop instead of using a feature branch, which it turns out we're not really meant to use.

Sorry for the noise.

### CODE REVIEW

File: src/include/wiredtiger.in [hunk 1]
```diff
@@ -7477,8 +7477,15 @@ typedef enum {
...
-    WT_VERBOSE_DEBUG        /*!< Low severity messages, useful for debugging purposes. */
+    WT_VERBOSE_DEBUG_1,     /*!< Low severity messages, useful for debugging purposes. This is
+                            the default level when debugging. */
+    WT_VERBOSE_DEBUG_2,     /*!< Low severity messages, an increase in verbosity from
+                            the previous level. */
+    WT_VERBOSE_DEBUG_3,     /*!< Low severity messages. */
+    WT_VERBOSE_DEBUG_4,     /*!< Low severity messages. */
+    WT_VERBOSE_DEBUG_5      /*!< Lowest severity messages. */
 } WT_VERBOSE_LEVEL;
+#define WT_VERBOSE_DEBUG WT_VERBOSE_DEBUG_1 /* Temporary, for compatibility. FIXME WT-9925. */
```
Comment (luke-pearson) [inline src/include/wiredtiger.in:7488]: Super nitpick: But, add a : after the "FIXME".

Comment (korteland) [reply src/include/wiredtiger.in]: thanks, done in 8c53af4.

**Review by luke-pearson — APPROVED** [review]
LGTM.

**Comment by korteland** [comment]
evergreen merge

**Pull request merged**